internal/pkg/service/cli/cmd/remote/create/table/dialog.go

```diff
@@ -25,9 +27,17 @@ const (
 
 type columnsDefinitionMethod int
 
-func AskCreateTable(args []string, branchKey keboola.BranchKey, allBuckets []*keboola.Bucket, d *dialog.Dialogs, f Flags) (table.Options, error) {
+func AskCreateTable(args []string, branchKey keboola.BranchKey, allBuckets []*keboola.Bucket, d *dialog.Dialogs, f Flags, backends []string) (table.Options, error) {
 	opts := table.Options{}
 
+	if f.OptionsFrom.IsSet() && f.ColumnsFrom.IsSet() && slices.Contains(backends, BackendBigQuery) {
```

Comment:
flag `options-from` have to be combined with `columns-from` flag, because BigQuery settings must have a specific column type.

Reply:
:one: `IsSet` should be used only to detect if a dialog should be displayed or not, do you remember the last incident?

`--options-from ""` shoudl mean "no options"

Reply:
> flag `options-from` have to be combined with `columns-from` flag, because BigQuery settings must have a specific column type.

You are right, but it should be an error, not shouldn't silent ignore it.